nucypher/policy/collections.py

```diff
@@ -163,6 +167,29 @@ def public_id(self):
         _id = keccak_digest(bytes(self._verifying_key) + bytes(self._hrac)).hex()
         return _id
 
+    def __serialize(self):
+        """
+        WAIT! This is probably not what you want. In 9/10 times, you'll
+        probably want to use `bytes(my_treasure_map)`. So... do that.
+        """
+        return self._public_signature + self._hrac + self._m.to_bytes(1, 'big') + self.nodes_as_bytes()
+
+    @classmethod
+    def __deserialize(cls, bytes_representation):
+        """
+        WAIT! This is probably not what you want. In 9/10 times, you'll probably
```

Comment:
Hmm - can you be more specific bout the condition in which you *do* want to use this method in the doctring?